src/utilities/http/fillInErrorWithDefaults.ts

```diff
@@ -8,7 +8,7 @@ export const fillInErrorWithDefaults = (error: Partial<HttpErrorResponseModel>,
     status: error.status || 0,
     message: error.message || 'Error requesting data',
     url: error.url || request.url,
-    raw: error.raw,
+    raw: null,
```

Comment:
https://github.com/zeit/next.js/blob/master/errors/circular-structure.md

circular-structure was happing on errors because I held onto the same error object that next.js gave me.